das/daser.go

```diff
@@ -90,5 +148,97 @@ func (d *DASer) sampling(ctx context.Context, sub header.Subscription) {
 		sampleTime := time.Since(startTime)
 		log.Infow("sampling successful", "height", h.Height, "hash", h.Hash(),
 			"square width", len(h.DAH.RowsRoots), "finished (s)", sampleTime.Seconds())
+
+		checkpoint = h.Height
```

Comment:
nit: I would rename this checkpoint to something like `height` or `sampleHeight` not mix it up with real catching up checkpoint

Reply:
resolved? https://github.com/celestiaorg/celestia-node/pull/473/commits/ad2975d10455ed92488bdbd81d5c30cf59e3a0bf